manifests/p/ppy/osu/2024.1219.2/ppy.osu.locale.zh-CN.yaml

```
@@ -19,6 +19,7 @@ ShortDescription: 节奏跃然指上
 Description: 一款免费的节奏游戏。节奏跃然指上。
 #Moniker: osu!
 Tags:
+  - 音游
```

```suggestion
- 音游
```

Sorry, automation error.